packages/generator-nitro/generators/app/templates/package.json

```diff
@@ -148,17 +154,17 @@
       "prettier --write",
       "npm run lint:data",
       "git add"
-    ],
+    ],<% if (options.compiler === 'js') { %>
     "src/**/*.{js,jsx}": [
       "prettier --write",
       "eslint",
       "git add"
-    ],
+    ],<% } %><% if (options.compiler === 'ts') { %>
```

Comment:
would handle it in `"src/**/*.{js,jsx}": [`
and only switch the extensions (config not needed)

Reply:
done